Transfer - Sample #06 - Connector Pull - build fails

The build fails with the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
org.gradle.api.InvalidUserDataException: Invalid catalog definition:
  - Problem: In version catalog libs, version reference 'okhttp' doesn't exist.
    
    Reason: Dependency 'com.squareup.okhttp3:okhttp-dnsoverhttps' references version 'okhttp' which doesn't exist.
    
    Possible solutions:
      1. Declare 'okhttp' in the catalog.
      2. Use one of the following existing versions: 'jackson' or 'jetbrainsAnnotation'.
    
    Please refer to https://docs.gradle.org/7.5.1/userguide/version_catalog_problems.html#undefined_version_reference for more details about this problem.
> Invalid catalog definition:
    - Problem: In version catalog libs, version reference 'okhttp' doesn't exist.
      
      Reason: Dependency 'com.squareup.okhttp3:okhttp-dnsoverhttps' references version 'okhttp' which doesn't exist.
      
      Possible solutions:
        1. Declare 'okhttp' in the catalog.
        2. Use one of the following existing versions: 'jackson' or 'jetbrainsAnnotation'.
      
      Please refer to https://docs.gradle.org/7.5.1/userguide/version_catalog_problems.html#undefined_version_reference for more details about this problem.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7s

```

Tested on Ubuntu 20.04 and Mac OS X 13.0

I am not a Java expert but it seems like a dependency issue. Requesting a quick fix for this issue.

--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -35,7 +35,7 @@ dependencyResolutionManagement {
     }
     versionCatalogs {
         create("libs") {
-            from("org.eclipse.edc:edc-versions:0.0.1-SNAPSHOT")
+            from("org.eclipse.edc:edc-versions:0.0.1-milestone-8")
             // this is not part of the published EDC Version Catalog, so we'll just "amend" it
             library(
                     "dnsOverHttps",

